README.md

```diff
@@ -213,6 +213,11 @@ Supported Options Reference
 
  NOTE that "never" is achieved by not setting any `Unattended-Upgrade::Mail`
 
+* `Unattended-Upgrade::Remove-Unused-Kernel-Packages` - boolean (default:True)
+ 
+ Remove excessive unused kernel-related packages before upgrading so that
```

Comment:
Thanks!
The phrasing "... to avoid filling /boot or the root file system." may be even better.